data/docs/instrumentation/manual-instrumentation/javascript/nodejs.mdx

```diff
@@ -0,0 +1,157 @@
+      span.addEvent('Event description') // Add an event with a description to the span
+      span.setAttribute('attributeKey', 'attributeValue') // Set an attribute on the span
+      span.setStatus({ code: SpanStatusCode.OK }) // Set the status of the span to OK
+    } catch (error) {
+      span.setStatus({ code: SpanStatusCode.ERROR, message: error.message }) // Set the status to ERROR if an exception occurs
+      span.recordException(error) // Record the exception in the span
+      throw error
+    } finally {
+      span.end() // End the span
+    }
+  })
+}
+```
+
+#### Running the Application
+
+Run your application normally, ensuring the tracing configuration is loaded first.
+
+#### Verifying the Manual Spans
+
+Check SigNoz for the manually created spans and verify the attributes and events are correctly logged.
+
+For more detailed instructions with a [sample app](https://github.com/SigNoz/opentelemetry-nodejs-example), refer to the complete article in the series: [Manual Instrumentation for Traces - OpenTelemetry NodeJS](https://signoz.io/opentelemetry/add-manual-span-to-traces-nodejs/)
+
+### Set Up Custom Metrics
+
+#### Setup Metric Collection
+
+Create a `telemetry.js` file and configure the `MeterProvider` with an `OTLPMetricExporter`:
+
+```jsx
+import { metrics } from '@opentelemetry/api'
+import {
+  MeterProvider,
+  PeriodicExportingMetricReader,
+  ConsoleMetricExporter,
+} from '@opentelemetry/sdk-metrics'
+import { Resource } from '@opentelemetry/resources'
+import {
+  SEMRESATTRS_SERVICE_NAME,
+  SEMRESATTRS_SERVICE_VERSION,
+} from '@opentelemetry/semantic-conventions'
+import { OTLPMetricExporter } from '@opentelemetry/exporter-metrics-otlp-http'
+
+// Define resource with service name
+const resource = Resource.default().merge(
+  new Resource({
+    [SEMRESATTRS_SERVICE_NAME]: 'your-service-name',
+  })
+)
+
+// Configure OTLP metrics exporter
+const otlpMetricsExporter = new OTLPMetricExporter({
+  url: 'http://otel-collector:4318/v1/metrics', // ensure the port is correct
+})
+
+const metricReader = new PeriodicExportingMetricReader({
+  exporter: otlpMetricsExporter,
+
+  // Default is 60000ms (60 seconds). Set to 10 seconds for demonstrative purposes only.
+  exportIntervalMillis: 10000,
+})
+
+// Initialize MeterProvider
+const myServiceMeterProvider = new MeterProvider({
+  resource: resource,
+  readers: [metricReader],
+})
+
+// Set this MeterProvider to be global to the app being instrumented.
+metrics.setGlobalMeterProvider(myServiceMeterProvider)
+```
+
+#### Collect Metrics in Business Logic
+
+Modify your business logic to measure and record metrics.
+
+```jsx
+import { performance } from 'perf_hooks'
+
+const meter = metrics.getMeter('your-service-name')
+
+async function yourFunction(parameter) {
+  const startTime = performance.now()
+  const yourMetric = meter.createHistogram('your-metric-name', {
+    description: 'Description of your metric',
```

Comment:
Can you please explain after the code snippet what `Description of your metric` refers to